wiki/Beatmap/Pattern/osu!/csj/en.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+tags:
+  - CSJ
+---
+
+# Cross Screen Jumps
+
+**Cross Screen Jumps** are a type of [Jumps](/wiki/Beatmap/Pattern/osu!/Jump) that occur when the distance between two [hit objects](/wiki/Gameplay/Hit_object) is big enough to go across most of the [playfield](/wiki/Client/Playfield).
```

Comment:
```suggestion
**Cross Screen Jumps** are a type of [jump](/wiki/Beatmap/Pattern/osu!/Jump) pattern that occur when the distance between two [hit objects](/wiki/Gameplay/Hit_object) is big enough to go across most of the [playfield](/wiki/Client/Playfield).
```

Reply:
"when the distance between two hit objects is big enough to go across most of the playfield." this could definitely be made more clear to the reader, unfortunately i'm not sure how to go about rewriting this